__tests__/account.test.ts

```diff
@@ -408,7 +406,7 @@ describe('deploy and test Wallet', () => {
       await provider.waitForTransaction(declareAccount.transaction_hash);
 
       const privateKey = stark.randomAddress();
-      starkKeyPub = getStarkKey(privateKey);
+      starkKeyPub = privateKey;
```

Comment:
Removing `getStarkKey` makes the tests not pass.

Reply:
It's my bad. Shouldn't remove that